privacy_partner_to_be_forgotten/tests/test_partner_anonymize.py

```
@@ -0,0 +1,369 @@
+        cls.user_stanley = cls.env.ref(
+            "privacy_partner_to_be_forgotten.user_stanley_marsh"
+        )
+
+        cls.test_user = cls.env["res.users"].create({
+            "name": "Test User",
+            "login": "test_user",
+            "email": "test_user@example.com",
+            "groups_id": [
+                (4, cls.env.ref("base.group_user").id),
+                (4, cls.env.ref("base.group_system").id),
+                (4, cls.env.ref("base.group_partner_manager").id),
```

Ruff (0.8.2)

Line too long (95 > 88)

(E501)